packages/wallet-sdk/README.md

```diff
@@ -0,0 +1,126 @@
+# Coinbase Wallet SDK
```

Comment:
npm publish requires reified README.md file. i tried symlink, but it didn't work.